website/screens/components/switch/code/examples/uncontrolled.ts

```diff
@@ -1,20 +1,44 @@
-import { DxcSwitch, DxcInset } from "@dxc-technology/halstack-react";
+import {
+  DxcSwitch,
+  DxcInset,
+  DxcFlex,
+  DxcButton,
+} from "@dxc-technology/halstack-react";
+import { useRef } from "react";
 
 const code = `() => {
+  const switchRef = useRef();
+
+  const handleSubmit = () => {
+    const switchEl = switchRef.current.getElementsByTagName("input")[0];
+    console.log(switchEl.value);
+  };
+
   const onChange = (newValue) => {
     console.log(newValue);
   };
 
   return (
     <DxcInset space="2rem">
-      <DxcSwitch label="Bluetooth" defaultChecked={true} onChange={onChange} />
+      <DxcFlex direction="column" gap="2rem">
+        <DxcSwitch
+          label="Bluetooth"
+          defaultChecked={true}
```

Comment:
The example lacks a `value` prop. Right now, when submitting, it will log an empty string in the console.

Reply:
Done